admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,381 @@
+[[admin-guide-node-problem-detector]]
+= Node Problem Detector
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+The Node Problem Detector monitors the health of your nodes
+by finding certain problems and reporting these problems to the API server.
+The detector runs as a daemonset on each node.
+
+[IMPORTANT]
+====
+The Node Problem Detector is a Technology Preview feature
+only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+The xref:../install/index.adoc#install-planning[installation] creates 
+a Node Problem Detector daemonset by default and creates a project for the detector, called `openshift-node-problem-detector`.
+
```

Comment:
@joelsmith Does the user need to do anything on master and/or on the nodes to start NPD? oc create the daemonset? Or, does it run by default?

Reply:
No, if they have run the installer with the variable `openshift_node_problem_detector_install=true` on the inventory, then the daemonset will be configured and the cluster will run NPD for them.  However, I don't see the installation via openshift-ansible documented anywhere.  Could we direct users to know that they need to set `openshift_node_problem_detector_install` to `true` for their install?

Reply:
Which section does `openshift_node_problem_detector_install=true` in the inventory file?